packages/eslint-plugin/src/rules/keyword-spacing.ts

```diff
@@ -0,0 +1,272 @@
+import baseRule from 'eslint/lib/rules/keyword-spacing';
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { isTokenOnSameLine } from '../util/astUtils';
+import keywords from 'eslint/lib/rules/utils/keywords';
+
+import * as util from '../util';
+
+export type Option = Partial<{
```

Comment:
This should be `Options` (it's what CI is currently failing on, since it's referenced further down)